### DISCUSSION TITLE
582

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#963] [Comments]
# 582

프론트엔드 모니터링을 보다보면 다음과 같이 Script error. 만 나올때가 있습니다. 나머지 에러들은 정확하게 에러가 노출되지만, 특정 케이스에 한해서 Script error. 만 나오게 되는 것인데요. 왜 모니터링에서는 Script error. 만 나오며, 이를 해결하기 위해선 어떻게 해야하는지 알아보겠습니다. 1. 배경 소개 예를 들어 다음과 같은 sdk.js 파일을 사용하는 웹 페이지가 있다고 가정해봅니다. sdk.js function foo() { console.log($('#a').text()); } index.html test 코드는 간단합니다. sdk.js 와 jquery를 호출하고, 이를 index.html이 sdk.js의 foo funciton을 호출 sdk.js는 jquery를 의..

https://jojoldu.tistory.com/582

**Comment by jws1837** [comment]
지금 이상황을 겪고 있는데 역시나 향로님이 글을 적어주셨었네요. 감사합니다. 도움됐습니다.
